client/src/components/menubar/index.js

```diff
@@ -0,0 +1,693 @@
+// jshint esversion: 6
+import React from "react";
+import { connect } from "react-redux";
+import {
+  Button,
+  AnchorButton,
+  Tooltip,
+  Popover,
+  Menu,
+  MenuItem,
+  Position,
+  NumericInput,
+  Divider,
+  Icon,
+  RadioGroup,
+  Radio
+} from "@blueprintjs/core";
+import Logo from "../framework/logo";
+import { World } from "../../util/stateManager";
+import * as globals from "../../globals";
+import actions from "../../actions";
+import CellSetButton from "./cellSetButtons";
+
+@connect(state => ({
+  differential: state.differential,
+  world: state.world,
+  loading: state.controls.loading,
+  datasetTitle: state.config?.displayNames?.dataset,
```

Comment:
this will be null if not defined, which seems wrong given how it is later used.    Probably should be defaulted to an empty string:  `datasetTitle: state.config?.displayNames?.dataset ?? ""`

Reply:
Indeed, when it was omitted we had vnull